os/booting-on-google-compute-engine.md

```diff
@@ -104,6 +108,34 @@ gcloud compute ssh --zone us-central1-a core@<instance-name>
 
 Users other than `core`, which are set up by the GCE account manager, may not be a member of required groups. If you have issues, try running commands such as `journalctl` with sudo.
 
+### Using OS Login
+
+You can log in using your google account on instances with OS Login enabled. OS Login needs to be [enabled in the GCE console](https://cloud.google.com/compute/docs/instances/managing-instance-access#enable_oslogin) and on the instance. It is enabled by default on the instance for nodes provisioned with Container Linux 1849.0.0 or later. Once enabled, you can log into your Container Linux instances using:
```

Comment:
s/google/Google/
s/using/using the following/ (or similar)

Reply:
I disagree with `using the following`. It also matches the wording in the above paragraph

Reply:
Just "using:" feels like a bit of a sentence fragment to me, but if it matches the rest of the document then I guess that's what matters.